### PR TITLE
Chores/backport k8s

### PR DESCRIPTION
**Pull request by armandfardeau**
#### :tophat: Description
*Please describe your pull request.*

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #?
- Fixes #?
- [Notion card]()

#### Testing
*Describe the best way to test or validate your PR.*

Example:
* Log in as admin
* Access Backoffice
* Go to organization settings
* See ...

#### Tasks
- [ ] Add specs
- [ ] Add note about overrides in OVERLOADS.md

### :camera: Screenshots
*Please add screenshots of the changes you're proposing if related to the UI*

### CODE REVIEW

File: .github/workflows/ci_cd.yml [hunk 1]
```diff
@@ -1,5 +1,13 @@
 name: "CI/CD"
-on: [push]
+on:
```
Comment (Quentinchampenois) [inline .github/workflows/ci_cd.yml:2]: Do not merge CIs

File: config/initializers/decidim.rb [hunk 2]
```diff
@@ -1,19 +1,29 @@
...
+
 Decidim.configure do |config|
-  config.skip_first_login_authorization = ENV["SKIP_FIRST_LOGIN_AUTHORIZATION"] ? ActiveRecord::Type::Boolean.new.cast(ENV["SKIP_FIRST_LOGIN_AUTHORIZATION"]) : true
   config.application_name = "OSP Agora"
   config.mailer_sender = "OSP Agora <ne-pas-repondre@opensourcepolitics.eu>"
 
   # Change these lines to set your preferred locales
-  config.default_locale = :nl
-  config.available_locales = [:en, :fr, :nl]
+  if Rails.env.production?
```
Comment (Quentinchampenois) [inline config/initializers/decidim.rb:11]: Do not change locales

File: config/initializers/decidim.rb [hunk 3]
```diff
@@ -89,6 +99,38 @@
   end
 
   config.base_uploads_path = "#{ENV["HEROKU_APP_NAME"]}/" if ENV["HEROKU_APP_NAME"].present?
+
+  # Machine Translation Configuration
```
Comment (Quentinchampenois) [inline config/initializers/decidim.rb:103]: Remove machine translations

File: config/initializers/decidim.rb [hunk 4]
```diff
@@ -89,6 +99,38 @@
...
+  autoload :ReminderRegistry, "decidim/reminder_registry"
+  autoload :ReminderManifest, "decidim/reminder_manifest"
+  autoload :ManifestMessages, "decidim/manifest_messages"
+
+  def self.reminders_registry
+    @reminders_registry ||= Decidim::ReminderRegistry.new
+  end
+end
+
+Decidim.reminders_registry.register(:orders) do |reminder_registry|
```
Comment (Quentinchampenois) [inline config/initializers/decidim.rb:121]: Remove reminders registry

File: config/locales/fr.yml [hunk 5]
```diff
@@ -9,19 +9,26 @@ fr:
...
         private_space: Espace privé
   decidim:
-    accessibility:
-      skip_button: Passer au contenu principal
     admin:
       participatory_space_private_users:
         create:
           error: Erreur
           success: Succès
+      reminders:
```
Comment (Quentinchampenois) [inline config/locales/fr.yml:17]: Remove key reminder, budgets, anonymous_user, omniauth FC, devise

Add removed translations

File: config/secrets.yml [hunk 6]
```diff
@@ -11,25 +11,50 @@
...
+        period: <%= ENV["THROTTLING_PERIOD"]&.to_i || 60 %>
+  scaleway:
+    id: <%= ENV["SCALEWAY_ID"] %>
+    token: <%= ENV["SCALEWAY_TOKEN"] %>
+    bucket_name: <%= ENV["SCALEWAY_BUCKET_NAME"] %>
+    endpoint: <%= ENV.fetch("OBJECTSTORE_S3_HOST", "s3.fr-par.scw.cloud") %>
   helpscout:
     enabled: <%= !ENV["HELP_SCOUT_BEACON_ID_MAIN"].blank? %>
     main: <%= ENV["HELP_SCOUT_BEACON_ID_MAIN"] %>
     fallback: <%= ENV["HELP_SCOUT_BEACON_ID_FALLBACK"] %>
```
Comment (Quentinchampenois) [inline config/secrets.yml:32]: Re add hotjar

Omniauth cultuur_conenct

File: config/secrets.yml [hunk 7]
```diff
@@ -40,6 +65,11 @@ default: &default
     server: <%= ENV["ETHERPAD_SERVER"] %>
     api_key: <%= ENV["ETHERPAD_API_KEY"] %>
     api_version: "1.2.1"
+  translator:
```
Comment (Quentinchampenois) [inline config/secrets.yml:68]: Remove translator
Added SSOs

**Review by paulinebessoles — APPROVED** [review]

**Pull request merged**